.devcontainer/devcontainer.json

```diff
@@ -69,7 +70,7 @@
 			]
 		}
 	},
-	"postCreateCommand": "git config --global --add safe.directory ${containerWorkspaceFolder} && rustup component add rls rust-analysis rust-src rustfmt clippy",
+	"postCreateCommand": "git config --global --add safe.directory ${containerWorkspaceFolder} && rustup component add rls rust-analysis rust-src rustfmt clippy && cargo install cargo-sort just",
```

Comment:
Ideally I would like the dev containers to still be quick to get up and running for everyone, not just people who are looking to program in Rust. Installing cargo packages during the `postCreateCommand` slows this down.

I would suggest the following changes:

1. Install just using the pre-built binaries (e.g. https://just.systems/man/en/chapter_5.html)
2. Move the installation of `cargo-sort` to a task inside the Justfile, e.g. `just setup-dev` (or something like that)...so Rust developers can install those dependencies if needed, but people wanting to run the integration tests don't need to.

Reply:
I just reviewed the just file again, and there is already a task to install `cargo-sort`:

```sh
just install-tools
```